test/verify/check-system-journal

```diff
@@ -329,6 +329,31 @@ ExecStart=/bin/sh -c 'sleep 5; for s in $(seq 10); do echo SLOW; sleep 0.1; done
 
         self.allow_journal_messages(".*Data from the specified boot .* is not available: No such boot ID in journal.*")
 
+    def testIdentifierReloading(self):
+        b = self.browser
+        m = self.machine
+        tag_error = "check-error-inside"
+        tag_info = "check-info-inside"
+        m.execute(f"logger -p user.err --tag {tag_error} {tag_error.upper()}")
+        m.execute(f"logger -p 5 --tag {tag_info} {tag_info.upper()}")
+
+        self.login_and_go("/system/logs")
+        self.browser.select_PF4("#journal-prio-menu", "Info and above")
+        b.wait_visible(f"#journal-box .cockpit-logline .cockpit-log-message:contains('{tag_error.upper()}')")
+        b.wait_visible(f"#journal-box .cockpit-logline .cockpit-log-message:contains('{tag_info.upper()}')")
+
+        self.browser.select_PF4("#journal-prio-menu", "Error and above")
+        b.reload()
```

Comment:
Shouldn;t it work without this?

Reply:
It will work without this, but it enforces the state. It will pass also now withhout fix I think. it depends what is default  priority filter. Theoretically it could be simplified by using `.go("#/?priority=debug")`

Reply:
So you change priority to "Error and above" and then reload the page.

> but it enforces the state

What state? If you wan't to enforce something, please do so by checking some element are present or not present.

> Theoretically it could be simplified by using .go("#/?priority=debug")

So you change to "Error and above" and then go to `Debug`?

Reply:
Ah, you want to verify that item is hidden and then shown. For that do something like:

1. Change to error and above
2. Make sure you see the error-item
3. Make sure you don't see the info-item
4. Change to info and above
5. Make sure you see the info-item
6. Make sure you see the error-item

Reply:
yes, it is more less exactly what I'm doing, using select instead of just present. to prove it is also selectable.

Reply:
To avoid reloading you can first login, switch to error and above, then generate those two messages, check one identifier is present, change priority and check both identifiers are present. That should be simple. And with maybe one or two lines you can do this in the current test as well

Reply:
yes, this is more less my case what I've done, but it is done in vice-versa order. but should prove same.

Reply:
> To avoid reloading you can first login, switch to error and above, then generate those two messages, check one identifier is present, change priority and check both identifiers are present. That should be simple. And with maybe one or two lines you can do this in the current test as well

Reloading here is to ensure the proper state, otherwise it can happen to pass also in current workflow. reload ensures, that it does not met current behaviour. but you are right I can avoid reloading and do it as you suggested.